ras/hwinfo.py

```diff
@@ -49,17 +49,27 @@ def test_list(self):
             cmd = "hwinfo %s" % list_item
             self.run_cmd(cmd)
 
-    def test_disk(self):
+    def test_listmd(self):
```

Comment:
Suggesting can we have function `test_options` ?

Reply:
Renamed test_list to test_list_options as it makes more sense.
In the test-case, all the list options are being testing, hence renamed as suggested.
Thanks.